e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -554,6 +537,77 @@ describe("scenarios > collection defaults", () => {
           // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
           cy.findByText("Undo").should("not.exist");
         });
+
+        it("moving collections should disable moving into any of the moving collections", () => {
```

Comment:
this is one of the major benefits of the new UI: we disable moving collections into themselves in the UI, rather than waiting for the API to reject the request